editor/project_manager/project_dialog.cpp

```diff
@@ -1024,8 +1024,14 @@ ProjectDialog::ProjectDialog() {
 	add_child(fdialog_install);
 
 	project_name->connect(SceneStringName(text_changed), callable_mp(this, &ProjectDialog::_project_name_changed).unbind(1));
+	project_name->connect(SceneStringName(text_submitted), callable_mp(this, &ProjectDialog::ok_pressed).unbind(1));
```

Comment:
I would recommend to rather use the `register_text_enter` method for those three `LineEdit`s, e.g. here just call `register_text_enter(project_name);`.

This will effectively do the same, but also the disablement of the ok button is checked, so that `ok_pressed` is only called when it is enabled.

Reply:
That makes more sense indeed. I tested it and it works, thanks for the tip :slightly_smiling_face: